packages/shared/lib/core/profile/actions/active-profile/logout.ts

```diff
@@ -42,6 +44,14 @@ export function logout(clearActiveProfile: boolean = false, _lockStronghold: boo
         }
         resetDashboardState()
         resetRouters()
+
+        const manager = get(profileManager)
+
+        // start background sync
+        manager.startBackgroundSync()
```

Comment:
1. It is a typo. It should `stopBackgroundSync()`;
2. I think at this point we no longer have the manager instance available, so we need to make sure we move this slightly higher up where we still have access to the manager instance.

Reply:
Fixed in https://github.com/iotaledger/firefly/pull/3427/commits/55e24a8468db2298f52586ee4c03312d0b9bcc86